src/login/loginctl.c

```diff
@@ -572,6 +574,14 @@ static int print_session_status_info(sd_bus *bus, const char *path, bool *new_li
         if (i.state)
                 printf("\t   State: %s\n", i.state);
 
+        if (i.idle_hint && timestamp_is_set(i.idle_hint_timestamp.realtime))
+                printf("\t    Idle: %s since %s (%s)\n",
+                       yes_no(i.idle_hint),
+                       FORMAT_TIMESTAMP(i.idle_hint_timestamp.realtime),
+                       FORMAT_TIMESTAMP_RELATIVE(i.idle_hint_timestamp.realtime));
```

Comment:
This introduces a slight discrepancy between `list-sessions` and `show-sessions`, as `list-sessions` uses monotonic timestamp. IMHO the monotonic timestamp is more appropriate for _relative_ time; think about calling the the command right after a system time change.

Reply:
We use `realtime` for `Since :` too. Also, I just noticed that `TABLE_TIMESTAMP_RELATIVE` actually takes realtime and derives relative time from that... I'll add a commit to fix that in `list-sessions`.